static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -20,252 +20,256 @@
-            float: none;
-            top: auto;
-            left: auto;
-            right: auto;
-            bottom: auto;
-        }
-        &__play-button {
-            position: relative;
-            width: 92px;
-            height: 92px;
-            transform: none;
-            &:hover {
-                transform: none;
-            }
-            &:before {
-                width: 100%;
-                height: 100%;
-                border-radius: 100%;
-            }
-            &:after {
-                display: none;
-            }
-            svg {
-                width: 46px;
-                height: 39px;
-                fill: $multimedia-main-1;
-                position: absolute;
-                top: 50%;
-                left: 50%;
-                transform: translateX(-42%) translateY(-50%);
-            }
-        }
-        &__bottom-bar__duration {
-            position: relative;
-            -webkit-font-smoothing: antialiased;
-            font-size: 18px;
-            line-height: 34px;
-            left: 6px;
-            height: 32px;
-            padding: 0 12px;
-            background-color: rgba(51, 51, 51, .7);
-            border-radius: 32px;
-            &:before {
-                display: none;
-            }
-        }
+.youtube-media-atom__iframe.youtube__video-ended.youtube__video-started ~ .end-slate-container {
+    @include mq($from: tablet) {
+        display: block;
     }
+}
+
+.youtube-media-atom__overlay.vjs-big-play-button {
+    pointer-events: none;
+}
+
+.youtube-media-atom__overlay {
+    background-size: cover;
+    /**
+     * Fixed 1px line appearing above overlay occasionally.
+     * http://stackoverflow.com/questions/38695492/1px-glitch-appearing-below-image-when-browser-re-sizes-background-image
+     * http://stackoverflow.com/questions/14376328/background-sizecover-1px-whitespace-to-the-left
+    **/
+    background-position: 49% 49%;
+    background-repeat: no-repeat;
+    z-index: 1;
+    text-align: center;
+}
+
+.youtube-media-atom__overlay .video-overlay {
+    text-align: initial;
+}
+
+.youtube-media-atom__play-button.vjs-control-text {
+    overflow: hidden !important;
+    display: inline-block;
+    z-index: 2;
+}
+
+.youtube-media-atom__bottom-bar {
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 34px;
+    z-index: 2;
+}
+
+.youtube-media-atom__bottom-bar::before,
+.youtube-media-atom__bottom-bar::after {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    left: 0;
+    z-index: -1;
+}
+
+.youtube-media-atom__bottom-bar::before {
+    content: '';
+    opacity: .1;
+    background: $neutral-1;
+}
+
+.youtube-media-atom__bottom-bar::after {
+    @include simple-gradient(rgba(51, 51, 51, 0), rgba(51, 51, 51, .5));
+    content: '';
+    opacity: .7;
+}
+
+.youtube-media-atom__bottom-bar__icon {
+    float: left;
+    height: 100%;
+    width: 36px;
+}
+
+.youtube-media-atom__bottom-bar__icon svg {
```

Comment:
Is this svg generated in the scala template? If so, it will have the classname `youtube-media-atom__bottom-bar__icon__svg` and not have to be nested